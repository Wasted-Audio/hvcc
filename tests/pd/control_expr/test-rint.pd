#N canvas 827 239 734 565 12;
#X obj 40 30 loadbang;
#X obj 40 240 print;
#X msg 40 140 2.3;
#X obj 40 105 t b b b;
#X msg 101 140 -3.1;
#X obj 40 207 expr rint($f1);
#X msg 163 140 5.7;
#X obj 163 240 print;
#X obj 286 240 print;
#X obj 163 207 expr round($f1);
#X obj 286 207 expr nearbyint($f1);
#X obj 40 63 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X connect 0 0 11 0;
#X connect 2 0 5 0;
#X connect 2 0 9 0;
#X connect 2 0 10 0;
#X connect 3 0 2 0;
#X connect 3 1 4 0;
#X connect 3 2 6 0;
#X connect 4 0 5 0;
#X connect 4 0 9 0;
#X connect 4 0 10 0;
#X connect 5 0 1 0;
#X connect 6 0 5 0;
#X connect 6 0 9 0;
#X connect 6 0 10 0;
#X connect 9 0 7 0;
#X connect 10 0 8 0;
#X connect 11 0 3 0;
