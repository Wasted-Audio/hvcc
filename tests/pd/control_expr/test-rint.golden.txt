[@ 0.000] print: 6
[@ 0.000] print: 6
[@ 0.000] print: 6
[@ 0.000] print: -3
[@ 0.000] print: -3
[@ 0.000] print: -3
[@ 0.000] print: 2
[@ 0.000] print: 2
[@ 0.000] print: 2
